pkg/plugins/golang/v3/init.go

```diff
@@ -35,8 +35,8 @@ import (
 
 // Variables and function to check Go version requirements.
 var (
-	goVerMin = golang.MustParse("go1.17")
-	goVerMax = golang.MustParse("go1.18")
+	goVerMin = golang.MustParse("go1.18")
```

Comment:
Could we not change the min?
We officially will support go 1.18 as always.
However, have we any reason to believe that the CLI will not provide the scaffolds currently with 1.17?
No. So, I do not think that we need to force the go 1.18 in this case

Reply:
ok